pymc/tests/distributions/test_continuous.py

```diff
@@ -108,6 +108,18 @@ def test_lower_bounded(self):
         assert lower_interval.value == -2
         assert upper_interval is None
 
+    def test_bounded_value(self):
+        with pm.Model() as model:
+            dist = pm.TruncatedNormal(
+                'bounded_both', mu=1, sigma=2, lower=0, upper=3
+            )
+            value = pm.Deterministic('value', pm.logp(dist, [-2., 1., 4.]))
+
+        values = model.compile_fn(value)({})
+        assert np.isinf(values[0])
+        assert np.isfinite(values[1])
+        assert np.isinf(values[2])
```

Comment:
We don't need a model for this:

```suggestion
        dist = pm.TruncatedNormal("bounded_both", mu=1, sigma=2, lower=0, upper=3)
        logp = pm.logp(dist, [-2., 1., 4.]).eval()
        assert np.isinf(logp[0])
        assert np.isfinite(logp[1])
        assert np.isinf(logp[2])
```

Also can you just put this inside this function: https://github.com/pymc-devs/pymc/blob/c53cd2f5653f46c59206c66acd1fc1bc0ebf9e77/pymc/tests/distributions/test_continuous.py#L828

This class is about the transform more generally, not TruncatedNormal in specific

Reply:
Ah, right - moved it!